pkg/apis/velero/v1/backup.go

```diff
@@ -59,6 +59,15 @@ type BackupSpec struct {
 	// +nullable
 	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty"`
 
+	// OrLabelSelectors is list of metav1.LabelSelector to filter with
+	// when adding individual objects to the backup. If multiple provided
+	// they will be joined by the OR operator. LabelSelector as well as
+	// OrLabelSelectors cannot co-exist in backup request, only one of them
+	// can be used.
+	// +optional
+	// +nullable
+	OrLabelSelectors []*metav1.LabelSelector `json:"orLabelSelectors,omitempty"`
```

Comment:
Why not `*[]metav1.LabelSelector `? This means OrLabelSelectors can be nil, but if it is not nil, you don't have to check again that each OrLabelSelectors[i] is not nil.
```suggestion
	OrLabelSelectors *[]metav1.LabelSelector `json:"orLabelSelectors,omitempty"`
```

Reply:
on second thought...  this would nullable already.
```suggestion
	OrLabelSelectors []metav1.LabelSelector `json:"orLabelSelectors,omitempty"`
```